client/components/suggestion-search/index.jsx

```diff
@@ -20,6 +20,7 @@ class SuggestionSearch extends Component {
 		onChange: PropTypes.func,
 		suggestions: PropTypes.array,
 		value: PropTypes.string,
+		limit: PropTypes.oneOfType( PropTypes.bool, PropTypes.number ),
```

Comment:
Do we need to accommodate two prop types? If the prop is passed down (and its value is a number > 0) it should be enough (?)

Reply:
+1. We could either go `limit = 0` as unlimited, or leave it as an optional prop :)

Reply:
Good suggestion. I have a WP-wired brain that loves multiple typing in args and returns :)